### PR TITLE
Initial style fixes

### PR DESCRIPTION
**Pull request by vincit-matu**
Should fix styles for all current pages except the apartment details page (at least far enough that they don't appear broken).

### CODE REVIEW

File: frontend/src/features/housingCompany/HousingCompanyCreatePage.tsx [hunk 1]
```diff
@@ -142,6 +145,7 @@ const HousingCompanyCreatePage = (): JSX.Element => {
                     inputType="date"
                     label="Myyntihintaluettelon vahvistamispäivä"
                     fieldPath="sales_price_catalogue_confirmation_date"
+                    required
```
Comment (toikarin-vincit) [inline frontend/src/features/housingCompany/HousingCompanyCreatePage.tsx:148]: this is not actually required. it's OK this is sent as `null` but not as blank string `""`

File: frontend/src/styles/layout/_page.sass [hunk 2]
```diff
@@ -16,127 +16,231 @@
...
-          grid-column: 1
           font-size: $fontsize-heading-l
-          font-weight: 600
-          width: 90px
+          letter-spacing: -1px
+          text-transform: uppercase
+          width: calc(90px)
+          line-height: 1.25
+        //text-align: right
+        //align-items: center
```
Comment (toikarin-vincit) [inline frontend/src/styles/layout/_page.sass:83]: remove commented lines?

Comment (vincit-matu) [reply frontend/src/styles/layout/_page.sass]: Oops 😅 - leftovers from when I was trying to find a nice solution to styling it. Commented lines removed.

File: frontend/src/features/housingCompany/HousingCompanyCreatePage.tsx (hunk 1 above)
Comment (vincit-matu) [reply frontend/src/features/housingCompany/HousingCompanyCreatePage.tsx]: Initial value is now set to `null`, and removed the `required` attribute from the field in question.

Comment (toikarin-vincit) [reply frontend/src/features/housingCompany/HousingCompanyCreatePage.tsx]: works :+1:

**Review by toikarin-vincit — APPROVED** [review]
LGTM

**Pull request merged**